ocm/util.py

```diff
@@ -0,0 +1,58 @@
+import ocm
+
+
+def as_component(
+    component: ocm.Component | ocm.ComponentDescriptor,
+    /,
+):
+    if isinstance(component, ocm.ComponentDescriptor):
+        component = component.component
```

Comment:
This `component` value is actually never returned (instead the `ValueError` will be raised).

Reply:
are you sure? in line 10, there is an `if` (not an `elif`)

However, we can save an unnecessary invocation of `isinstance` by using `return`, so I will change it